src/pybind/mgr/dashboard/rbd_ls.py

```diff
@@ -2,7 +2,47 @@
 import rados
 import rbd
 from remote_view_cache import RemoteViewCache
+from types import VersionedSyncObject
 
+class RbdPoolLs(RemoteViewCache):
+    def __init__(self, module_inst, osd_pools):
+        super(RbdPoolLs, self).__init__(module_inst)
+        self.osd_pools = osd_pools
+        self.rbd_pools = []
+
+    def _get(self):
+        from mgr_module import ceph_state
+        ctx_capsule = ceph_state.get_context()
+
+        self.log.info("Constructing Rados")
+        cluster = rados.Rados(context=ctx_capsule)
```

Comment:
when I wrote the rbd_ls.py stuff it was the only place spinning up a rados.Rados -- now that we're using it elsewhere, the instance should probably be shared